app/views/system_notifications/_list.html.erb

```diff
@@ -0,0 +1,3 @@
+<% notifications.each_with_index do |notification, index| %>
+  <%= render 'notification.html.erb', notification: notification %>
```

Comment:
Change to `<%= render partial: "notification", locals: { notification: notification } %>`.